src/context/notes-context.js

```diff
@@ -0,0 +1,80 @@
+import axios from "axios";
+import { createContext, useContext, useState } from "react";
+
+const NotesContext = createContext();
+
+const NotesProvider = ({ children }) => {
+  const [notes, setNotes] = useState([]);
+
+  const getNotes = async (token) => {
+    try {
+      const response = await axios.get("/api/notes", {
+        headers: { authorization: token },
+      });
+
+      setNotes(response.data.notes);
+      console.log(response);
```

Comment:
Remove console logs if you don't need them anymore.

Reply:
Yupp thanks for pointing that out